src/lib.rs

```diff
@@ -76,4 +76,27 @@ impl RustBoy {
     pub fn get_mem_at(&self, loc: u16) -> u8 {
         self.cpu.get_mem_at(loc)
     }
-}
\ No newline at end of file
+}
+
+use std::os::raw::c_char;
```

Comment:
@coopersimon Should these ideally be at the top of the file?

Reply:
I think either at the top or inside the function where you are using them. But since it is one of the arguments, I would stick it at the top.